packages/chaire-lib-backend/src/api/captcha.routes.ts

```diff
@@ -22,14 +22,16 @@ export interface CaptchaMiddlewareOptions {
     headerName?: string; // Header to look for
     errorStatusCode?: number; // Status code to return on failure (default: 403)
     errorMessage?: string; // Error message to return
+    keepToken?: boolean; // Whether to re-use or delete the token after validation
 }
 
-const defaultMiddlewareOptions: CaptchaMiddlewareOptions = {
+const defaultMiddlewareOptions: Required<CaptchaMiddlewareOptions> = {
     tokenField: 'captchaToken',
     queryParam: 'captchaToken',
     headerName: 'X-Captcha-Token',
     errorStatusCode: 403,
-    errorMessage: 'Captcha validation failed'
+    errorMessage: 'Captcha validation failed',
+    keepToken: false // Default to re-using the token
```

Comment:
Need to update the comment after switching the default. 
Also while you are at it, could you add a comment to that explain the impact of setting it to true. 
Took me a bit of time to fully grasp it. 
If I understand properly, basically the idea, when set to true, is that the captcha validation will be valid for the lifetime of the session (or until it times out)